client/signup/config/steps-pure.js

```diff
@@ -723,7 +723,10 @@ export function generateSteps( {
 		},
 		'difm-page-picker': {
 			stepName: 'difm-page-picker',
-			providesDependencies: [],
+			providesDependencies: [ 'contentPages' ],
```

Comment:
Suggestion: Can we rename this to `selectedPageTitles` to make the meaning obvious?